src/aes.rs

```diff
@@ -69,6 +71,32 @@ pub fn decrypt(encrypted_data: &[u8], key: &H256, iv: &u128) -> Result<Vec<u8>,
     Ok(final_result)
 }
 
+// AES-256/GCM/Pkcs encryption.
+pub fn encrypt_gcm(data: &[u8], key: &H256, nonce: &u128) -> Result<Vec<u8>, Error> {
```

Comment:
gcm and gcm-siv are different. Please rename it to `encrypt_gcm_siv`.

Reply:
Modified : )